src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/nonassociations/NonAssociationsSynchronisationIntTest.kt

```diff
@@ -1,7 +1,12 @@
 package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.nonassociations
 
+import com.github.tomakehurst.wiremock.client.WireMock
+import com.github.tomakehurst.wiremock.client.WireMock.anyRequestedFor
```

Comment:
A lot of changes here - changing the inner classes hasn't helped with the readability, sorry